robot_skills/src/robot_skills/robot_part.py

```diff
@@ -176,7 +176,8 @@ def process_hardware_status(self, diagnostic_dict):
         diag = diagnostic_dict.get(self.__diagnostics_name, None)
 
         if not diag:
-            rospy.logwarn('no diagnostic msg received for the %s' % self.__diagnostics_name)
+            pass
+	    #rospy.logwarn('no diagnostic msg received for the %s' % self.__diagnostics_name)
```

Comment:
Why is this warning removed?

Reply:
Good catch. 

Because it was annoying, apparently. Needs a good looking after for Sergio, as that does miss some bodyparts that exist in code but not reality and thus things will complain at a high rate when diagnostics come in

Reply:
It was removed because it showed up with 100Hz frequency or something and everying else in your terminal becomes invisible at that point.

Reply:
Lets just make it a rospy.logwarnonce and be done with the issue on short term ;-)

Reply:
Logging Once
New in Lunar

Since Lunar, rospy supports writing log messages only once after it is spawned. The output will print a message once by `rospy.log*_once(msg)`.

```
 while True:
    rospy.loginfo_once("This message will print only once")
```

Reply:
`rospy.loginfo_throttle(Inf, "This message will print every infinity seconds, so basically just once")`

:-D

Reply:
Like this: 44eecf6bc5e2c6c4da7b23c26e75f53e6a420849 ??